hack/compare-gen-md.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+. $(dirname "$0")/common.sh
+
+MDFILE="TESTLIST.md"
+
+mv cnf-tests/${MDFILE} cnf-tests/${MDFILE}.old
+
+. $(dirname "$0")/generate-cnf-docs.sh
```

Comment:
Why not generating a new one in a different path (maybe _cache which is in .gitignore)?
This won't mess up with eventual changes, and will avoid us committing the .old file (which I will be guilty for sure :P)